src/doc/src/reference/unstable.md

```diff
@@ -641,6 +641,39 @@ cargo +nightly -Zunstable-options -Zconfig-include --config somefile.toml build
 
 CLI paths are relative to the current working directory.
 
+### host-config
+* Original Pull Request: [#9322](https://github.com/rust-lang/cargo/pull/9322)
+* Tracking Issue: [#3349](https://github.com/rust-lang/cargo/issues/3349)
```

Comment:
Can a dedicated tracking issue be opened for this instead of reusing a previous issue? Ideally we'd close all previous issues in favor of the tacking issue at that point.

Reply:
What should go into that issue? The existing issue seems to accurately describe this feature for the most part still.

Reply:
Personally I think opening a fresh tracking issue is best because it's an opportunity to, from a fresh perspective, summarize the historical discussions, describe the feature in accurate detail, and prevent others from having to read dozens of comments just to learn about an issue.

Reply:
Tracking issues #9452 and #9453 added.